.sync.yml

```diff
@@ -30,18 +29,12 @@ appveyor.yml:
 
   script: "\"bundle exec rake release_checks\""
   docker_sets:
-  - set: docker/ubuntu-16.04
-  - set: docker/ubuntu-14.04
-  - set: docker/debian-8
-  - set: docker/centos-7
+    - set: docker/ubuntu-18.04
+    - set: docker/ubuntu-16.04
+    - set: docker/ubuntu-14.04
+    - set: docker/debian-8
```

Comment:
Can you add debian 9, too?

Reply:
I have tried adding Debian 9 but I have been running into some problems with tests on that VM. It looks like puppet is incorrectly reporting the service provider on that platform. I will look into it once I have some more free time. But for now I did not add it because of failing tests.